doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -371,6 +380,117 @@ See the following table for a mapping between the devicetree input used by ``nrf
           /* gpio1 - P1.2 CTRLSEL = 3 */
           UICR_GPIO_PIN_CNF_CTRLSEL_SET(0x5f938400UL, 2, 3);
 
+UICR.WDTSTART
+=============
+
+UICR.WDTSTART configures the automatic start of a local watchdog timer before the application core is booted.
+This provides early system protection ensuring that the system can recover from early boot failures.
+
+The UICR.WDTSTART configuration consists of three sub-registers:
+
+UICR.WDTSTART.ENABLE
+  Controls whether the watchdog timer automatic start feature is enabled.
+
+UICR.WDTSTART.INSTANCE
+  Specifies which watchdog timer instance to configure and start.
+  Valid values are:
+
+  * ``WDT0`` - Use watchdog timer instance 0
+  * ``WDT1`` - Use watchdog timer instance 1
+
+UICR.WDTSTART.CRV
+  Sets the initial Counter Reload Value (CRV) for the watchdog timer.
+  This value determines the watchdog timeout period.
+  The CRV must be at least 15 (0xF) to ensure proper watchdog operation.
+
+UICR.SECURESTORAGE
+==================
+
+UICR.SECURESTORAGE configures the secure storage system used by |ISE| for persistent storage of cryptographic keys and trusted data.
+The secure storage is divided into separate partitions for different services and processor domains.
+The sum of all size configurations specified in ``UICR.SECURESTORAGE.*`` must be 4kB aligned.
+For more information, see :ref:`ug_nrf54h20_ironside_se_secure_storage`.
+
+The UICR.SECURESTORAGE configuration consists of the following sub-registers:
+
+UICR.SECURESTORAGE.ENABLE
+  Controls whether the secure storage feature is enabled.
+
+UICR.SECURESTORAGE.ADDRESS
+  Specifies the start address of the secure storage region in memory.
+  This address must be 4kB aligned and point to a valid memory region that can be used for secure storage.
+
+UICR.SECURESTORAGE.CRYPTO
+  Configures partition sizes for the Cryptographic service within the secure storage.
+
+  UICR.SECURESTORAGE.CRYPTO.APPLICATIONSIZE1KB
+    Sets the size of the APPLICATION domain partition for cryptographic storage, specified in 1 kiB blocks.
```

Comment:
```suggestion
    Sets the size of the ``APPLICATION`` domain partition for cryptographic storage, specified in 1 kiB blocks.
```
If we want to highlight this specific partition name, is should be between ticks. Otherwise, let's just use application domain